utils/uiTypes/VotePlugin.ts

```diff
@@ -148,12 +151,17 @@ export class VotingClient {
       this.clientType = VotingClientType.GatewayClient
       this.noClient = false
     }
+    if (this.client instanceof PythClient) {
+      this.clientType = VotingClientType.PythClient
+      this.noClient = false
+    }
   }
   withUpdateVoterWeightRecord = async (
     instructions: TransactionInstruction[],
     tokenOwnerRecord: PublicKey,
     type: UpdateVoterWeightRecordTypes,
-    createNftActionTicketIxs?: TransactionInstruction[]
+    createNftActionTicketIxs?: TransactionInstruction[],
+    voterWeightTarget?: PublicKey
```

Comment:
It's problematic that this parameter is not clearly pyth specific.
```suggestion
    pythVoterWeightTarget?: PublicKey
```

Reply:
renamed based on the suggestion